gzip/README.md

```diff
@@ -0,0 +1,5 @@
+# gzip 
+
+Use the `build.sh` script to download and build `gzip`.
+The resulting `gzip` binary is a static PIE executable loadable using the Unikraft ELF Loader.
+
```

Comment:
Add some examples on how to run the application.
Also add some sections for `building` and `running`, you can check an example [here](https://github.com/unikraft/static-pie-apps/pull/10/files#diff-217ed82f87b78b399e304b07a159b27dff327c0f83adf4a2fc30b03bcbf84b01).

Reply:
Added the sections